scripts/voqutil

```diff
@@ -0,0 +1,369 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import re
+import sys
+import ipaddress
+
+from natsort import natsorted
+from tabulate import tabulate
+from utilities_common import multi_asic as multi_asic_util
+from swsscommon import swsscommon
+
+# mock the redis for unit test purposes #
+try:
+    if os.environ["UTILITIES_UNIT_TESTING"] == "1":
+        modules_path = os.path.join(os.path.dirname(__file__), "..")
+        tests_path = os.path.join(modules_path, "tests")
+        sys.path.insert(0, modules_path)
+        sys.path.insert(0, tests_path)
+        import mock_tables.dbconnector
+        if os.environ["UTILITIES_UNIT_TESTING_TOPOLOGY"] == "multi_asic":
+            import mock_tables.mock_multi_asic
+            mock_tables.dbconnector.load_namespace_config()
+        else:
+            mock_tables.dbconnector.load_database_config()
+
+except KeyError:
+    pass
+
+
+# ========================== Common VOQ util logic ==========================
+
+SYSTEM_PORT_TABLE_PREFIX = "SYSTEM_PORT_TABLE:"
```

Comment:
Is the separator or table name defined in some python lib? If so, it would be great to import them, instead of hardcoding them.

Reply:
@vganesan-nokia can you address @ysmanman's comment ?

Reply:
Yes. I'll fix this if the table names are defined in some python lib.

Reply:
Fixed for table names